src/panels/config/zha/zha-clusters-data-table.ts

```diff
@@ -0,0 +1,91 @@
+import "../../../components/data-table/ha-data-table";
+import "../../../components/entity/ha-state-icon";
+
+import memoizeOne from "memoize-one";
+
+import {
+  LitElement,
+  html,
+  TemplateResult,
+  property,
+  customElement,
+} from "lit-element";
+import { HomeAssistant } from "../../../types";
+// tslint:disable-next-line
+import { DataTableColumnContainer } from "../../../components/data-table/ha-data-table";
+// tslint:disable-next-line
+import { Cluster } from "../../../data/zha";
+import { formatAsPaddedHex } from "./functions";
+
+export interface ClusterRowData extends Cluster {
+  cluster?: ClusterRowData;
+}
+
+@customElement("zha-clusters-data-table")
+export class ZHAClustersDataTable extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public narrow = false;
+  @property() public clusters: Cluster[] = [];
+
+  private _clusters = memoizeOne((clusters: Cluster[]) => {
+    let outputClusters: ClusterRowData[] = clusters;
+
+    outputClusters = outputClusters.map((cluster) => {
+      return {
+        ...cluster,
+        cluster_id: cluster.endpoint_id + "-" + cluster.id,
```

Comment:
What is it used for?

Reply:
the same cluster can be on multiple endpoints so this gives us a unique id for the cluster